sklearn/svm/tests/test_svm.py

```diff
@@ -560,6 +560,18 @@ def test_sparse_precomputed():
         assert "Sparse precomputed" in str(e)
 
 
+def test_sparse_fit_support_vectors_empty():
+    X_train = sparse.csr_matrix([[0, 1, 0, 0],
```

Comment:
Please add a note like: `# Regression test for #14893`.

Reply:
Done :)